general/lecture1/tasks/_2/Task2.java

```diff
@@ -0,0 +1,39 @@
+package general.lecture1.tasks._2;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a distance in meters and time in seconds and outputs a speed in meters/second and kilometers/hour.
+ * The result should be a floating point number.
+ **/
+public class Task2 {
+
+    public Double convertSpeedValueToMs(double distance, double time) {
```

Comment:
it's better to use primitive types (`double`) instead of wrapper objects (`Double`), except you don't have a specific reason for this (in a current case we can use primitive types)